dist/profile/templates/buildmaster/casc/clouds.yaml.erb

```diff
@@ -41,7 +41,7 @@ jenkins:
         preInstallSsh: false
         retentionStrategy:
           azureVMCloudRetentionStrategy:
-            idleTerminationMinutes: 5
+            idleTerminationMinutes: "<%= agent["idleTerminationMinutes"] %>"
```

Comment:
questionable if this needs to be configurable

Reply:
Same (the credential is different between ci.j, trusted.ci and others (both its id and the value), hence the need to configure it.)

Reply:
This value is directly related to the combination of instance size + cloud (billing is different) + the expected usage for builds.